packages/frontend/src/lib/BootcCurrentProgress.svelte

```diff
@@ -0,0 +1,68 @@
+<script lang="ts">
+import type { BootcBuildInfo, BootcBuildStatus } from '/@shared/src/models/bootc';
+import { bootcClient } from '../api/client';
+
```

Comment:
TBH I really don't like the progress column, feels like duplication with status and odd to have a badge here. I am ok merging for now though, I think once we have something like build names we could refactor or look at other options.